### DISCUSSION TITLE
Open Terrain Generator compatability

### DISCUSSION THREAD

**Discussion by PaxJaromeMalues** [EwyBoy/OreTweaker#84] [Ideas]
https://www.curseforge.com/minecraft/mc-mods/open-terrain-generator/files/all?filter-game-version=2020709689%3A6756

It would be really nice, if you could check if your mod actually already works via forge hand in hand with OTG or if it could be implemented to somehow work together.

**Comment by EwyBoy** [comment]
Would be cool, but I no longer update the 1.12 version of the mod.
